src/application.vala

```diff
@@ -232,24 +224,109 @@ public class Application : Gtk.Application {
         }
     }
 
-    private void show_qrcode() {
+    private void create_cert() {
+        GnuTLS.X509.PrivateKey key = GnuTLS.X509.PrivateKey.create();
+        key.generate(GnuTLS.PKAlgorithm.RSA, 2048);
+        //GnuTLS.X509.Certificate certi = GnuTLS.X509.Certificate.create();
+    }
+
+    private string? call_sytem_function (string[] args) {
```

Comment:
I do not like this function name, the name should describe what the function does, not how: I would say run_generate_cert_script